pydoctor/test/test_zopeinterface.py

```diff
@@ -160,13 +161,16 @@ class IMyInterface(interface.Interface):
     mod = fromText(src, modname='mod', systemcls=ZopeInterfaceSystem)
     text = mod.contents['IMyInterface'].contents['text']
     assert text.docstring == 'fun in a bun'
-    assert text.kind == "TextLine"
+    assert str(text.parsed_type.to_stan(NotFoundLinker())) == "Tag('code', children=[Tag('', children=['schema.TextLine'])])"
```

Comment:
In other tests we use the following approach:
```py
assert type2html(c) == '<code>str</code>'
```
I think that is a bit more robust against changes in the HTML generation process, as well as being easier to read.